docs/4_2_co-simulation_api.adoc

```diff
@@ -260,6 +260,8 @@ _For example, if the slave does not use the update formula for the <<independent
 
 It depends on the capabilities of the FMU which argument constellations and calling sequences are allowed (see <<fmi-for-co-simulation>>).
 
+Only <<fmi3DoStep>> can change the time inside a Co-Simulation FMU.
```

Comment:
would change this to something like
`Only <<fmi3DoStep>> can change the time of a Co-Simulation FMU from the outside (while time advances internally during a communication interval).`

Reply:
Agree, much clearer this way.